supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -265,7 +265,7 @@ A client requests a certificate from a web server. On receiving the certificate,
 [discrete]
 [[transactions]]
 ==== image:images/yes.png[yes] transactions subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "transactions" subsystem is used to configure options in the Transaction Manager. Write in lowercase in general text. Use "Transactions subsystem" when referring to the transactions subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _transactions_ subsystem is used to configure options in the Transaction Manager. Write in lowercase in general text. Use "Transactions subsystem" when referring to the transactions subsystem in titles and headings.
```

Comment:
_transactions_ subsystem_ to fit the title of the entry? 

Consider surrounding the last `transactions` entry in monospace to align with highlighting conventions for component names.

Reply:
I followed your lead in your PR #217, which didn't include "subsystem" in the italics. If you want to do that, we should do it throughout all of our PRs. wdyt?

Reply:
Yeah, I realized on the latest changes I made the title mentions the word "subsystem", so maybe placing the word in italics also would help? I could apply this to any changes that I've made also.

Reply:
Sounds good, I'll make the changes in my files as well!